pkg/config/config.go

```diff
@@ -39,7 +39,7 @@ type Flags struct {
 	// Version decides whether to just print the version and exit.
 	Version bool `long:"version" description:"print version and exit"`
 	// Config is the path to the config file
-	Config string `short:"c" long:"config" description:"path to config file" required:"true" default:"./config.yml"`
+	Config string `short:"c" long:"config" description:"path to config file" required:"true" default:"/etc/icingadb/icingadb.yml"`
```

Comment:
Packages install it to `/etc/icingadb/config.yml`:
* https://git.icinga.com/packaging/deb-icingadb/-/blob/4f37842d13ed170dd3b50d8f8dcd44611ae3ad76/debian/install#L4
* https://git.icinga.com/packaging/rpm-icingadb/-/blob/36775ec19a18c86c70547ba0727120f028fd30ab/icingadb.spec#L76